client/flutter/pubspec.lock

```diff
@@ -125,7 +125,7 @@ packages:
       name: pedantic
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "1.8.0+1"
+    version: "1.9.0"
```

Comment:
Why did this get bumped?

Reply:
Flutter tooling doing this automatically.